app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,33 @@
+class BookingsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @event = Event.find params[:event_id]
+
+    @bookings = @event.bookings
+  end
+
+  def create
+    @event = Event.find params[:event_id]
+    @applicant = User.find params[:applicant_id]
+
+    @event.booked_by! @applicant
+
+    redirect_to :back
+  end
+
+  def destroy
+    @booking = Booking.find(params[:id])
+    @booking.destroy
+
+    redirect_to :back
+  end
+
+  def accept
+    @booking = Booking.find(params[:booking_id])
+
+    @booking.event.accept_booking! @booking
```

Comment:
Seeing the same `@booking` twice in this line. Might be a sign that there's an instance method to add to `Booking` that could do all of this work in one call.